recipes/openvdb/8.1.0/conanfile.py

```diff
@@ -0,0 +1,205 @@
+    )
+    license = "MPL-2.0"
+    topics = ("voxel", "voxelizer", "volume-rendering", "fx")
+    homepage = "https://github.com/AcademySoftwareFoundation/openvdb"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_blosc": [True, False],
+        "with_zlib": [True, False],
+        "with_log4cplus": [True, False],
+        "simd": [None, "sse42", "avx"],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_blosc": True,
+        "with_zlib": True,
+        "with_log4cplus": False,
+        "simd": None,
+    }
+
+    _cmake = None
+
+    _compilers_min_version = {
+        "msvc": "19.10",
+        "Visual Studio": "15",  # Should we check toolset?
+        "gcc": "6.3.1",
+        "clang": "3.8",
+        "apple-clang": "3.8",
+        "intel": "17",
+    }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def _check_compilier_version(self):
+        compiler = str(self.settings.compiler)
+        version = tools.Version(self.settings.compiler.version)
+        if version < self._compilers_min_version[compiler]:
+            raise ConanInvalidConfiguration("%s requires a %s version greater than %s" % (self.name, compiler, self._compilers_min_version[compiler]))
+
+    def validate(self):
+        if self.options.shared:
+            if self.settings.compiler == "Visual Studio" and "MT" in self.settings.compiler.runtime:
+                raise ConanInvalidConfiguration("Visual Studio build for shared library with MT runtime is not supported")
+
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 14)
+        if self.settings.arch not in ("x86", "x86_64"):
+            if self.options.simd:
+                raise ConanInvalidConfiguration("Only intel architectures support SSE4 or AVX.")
+        self._check_compilier_version()
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("boost/1.76.0")
+        self.requires("tbb/2020.3")
+        if self.options.with_zlib:
+            self.requires("zlib/1.2.11")
+        if self.options.with_blosc:
+            self.requires("c-blosc/1.20.1")
+        if self.options.with_log4cplus:
+            self.requires("log4cplus/2.0.5")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+
```

Comment:
```suggestion

    def build_requirements(self):
        if self.settings.os == "Macos" and self.settings.arch == "armv8":
            # Attempting to use @rpath without CMAKE_SHARED_LIBRARY_RUNTIME_C_FLAG being
            # set. This could be because you are using a Mac OS X version less than 10.5
            # or because CMake's platform configuration is corrupt.
            self.build_requires("cmake/3.20.1")
```